### ISSUE TITLE
`security` tool fails when codesigning with 3.5.0

### ISSUE THREAD

**Issue by erikkerber** [bazelbuild/rules_apple#2443]
Our builds began seeing codesigning errors as of 3.5.0. Our best guess is something might have been introduced with #2438, but aren't quite sure what exactly about that change altered behavior.

```
security: cert import failed: UNIX[Operation not permitted]
security: problem decoding
...
File "/private/var/tmp/_bazel_buildkite/6357013eb1e6bdc4f3d11ac8e827aac3/sandbox/darwin-sandbox/20568/execroot/_main/bazel-out/darwin_arm64-opt-exec-ST-13d3ddad9198/bin/external/rules_apple~3.5.0/tools/imported_dynamic_framework_processor/imported_dynamic_framework_processor.runfiles/rules_apple~3.5.0/tools/codesigningtool/codesigningtool.py
...
subprocess.CalledProcessError: Command '['security', 'cms', '-D', '-i', 'App/Profiles/InHouse.mobileprovision']' returned non-zero exit status 1.
```

**Comment by krypt-lx** [comment]
Observed the same behaviour after dependencies upgrade yesterday, had to downgrade rules_apple to 3.4.0

**Comment by luispadron** [comment]
Fixed in #2444
